### PR TITLE
Loki: Fix misaligned derived fields settings

### PR DESCRIPTION
**Pull request by svennergr**
**What is this feature?**

The derived fields settings in Loki were misaligned.

**Why do we need this feature?**

Now:
image

Previously:
image

### CODE REVIEW

File: public/app/plugins/datasource/loki/configuration/DerivedField.tsx [hunk 1]
```diff
@@ -97,8 +99,9 @@ export const DerivedField = (props: Props) => {
         />
       </div>
 
-      <div className={styles.row}>
+      <div className="gf-form">
```
Comment (ivanahuckova) [inline public/app/plugins/datasource/loki/configuration/DerivedField.tsx:102]: I think we could even remove on line 96 to have same spacing across whole form. 
```
css`
            margin-left: 8px;
          `
```

**Review by ivanahuckova — COMMENTED** [review]
🤩 Left comment about redundant spacing for button.

File: public/app/plugins/datasource/loki/configuration/DerivedField.tsx (hunk 1 above)
Comment (svennergr) [reply public/app/plugins/datasource/loki/configuration/DerivedField.tsx:99]: Nice! Absolutely, removed in 9980f0e

**Review by ivanahuckova — APPROVED** [review]
LGTM

**Pull request merged**